conjur/cli.py

```diff
@@ -81,29 +119,30 @@ def run(self, *args):
         delete_policy_parser.add_argument('policy',
             help='File containing the YAML policy')
 
+        global_optional.add_argument('-h', '--help', action='help', help="Display help list")
+        global_optional.add_argument('-v', '--version', action='version',
+                                     help="Display version number",
+                                     version='%(prog)s version ' + __version__)
 
-        parser.add_argument('-v', '--version', action='version',
-            version='%(prog)s v' + __version__)
-
-        parser.add_argument('-l', '--url')
-        parser.add_argument('-a', '--account')
+        global_optional.add_argument('-l', '--url')
```

Comment:
Why don't subcommands actions have description too?

Reply:
these will be removed when we implement the `init` command